python/paddle/fluid/transpiler/__init__.py

```diff
@@ -15,6 +15,7 @@
 from __future__ import print_function
 
 from .distribute_transpiler import DistributeTranspiler, DistributeTranspilerConfig
+from .inference_transpiler import InferenceTranspiler
```

Comment:
If remove this line, errors:
```
[06:38:57][Step 1/1]     t = fluid.transpiler.InferenceTranspiler()
[06:38:57][Step 1/1] AttributeError: 'module' object has no attribute 'InferenceTranspiler'
```
http://ci.paddlepaddle.org/viewLog.html?buildId=19035&tab=buildLog&buildTypeId=Paddle_PrCi&logTab=tree&filter=all&_focus=17841